src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/DataGridViewCellAccessibleObjectTests.cs

```diff
@@ -1013,6 +1013,483 @@ public void DataGridViewCellAccessibleObject_GetPropertyValue_AutomationId_Retur
             Assert.False(dataGridView.IsHandleCreated);
         }
 
+        [WinFormsFact]
+        public void DataGridViewCellAccessibleObject_FragmentNavigate_Parent_ReturnsExpected()
+        {
+            using DataGridView dataGridView = new();
+            dataGridView.Columns.Add("Column 1", "Header text 1");
+            dataGridView.CreateControl(); // DataGridViewCellAccessibleObject.FragmentNavigate method needs Handle no provide non-empty results
+
+            var cellAccessibleObject = dataGridView.Rows[0].Cells[0].AccessibilityObject;
+            var rowAccessibleObject = dataGridView.Rows[0].AccessibilityObject;
+
+            Assert.Equal(rowAccessibleObject, cellAccessibleObject.FragmentNavigate(UiaCore.NavigateDirection.Parent));
+            Assert.True(dataGridView.IsHandleCreated);
+        }
+
+        [WinFormsFact]
+        public void DataGridViewCellAccessibleObject_FragmentNavigate_Parent_IsNull_IfOwningRowIsNull()
+        {
+            using var owner = new SubDataGridViewCell();
```

Comment:
In the new code let's stick with this init format:
```suggestion
            using SubDataGridViewCell owner = new();
```

Reply:
Fixed that in new commits.